src/tools/rbd_mirror/Throttler.cc

```diff
@@ -0,0 +1,204 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#include "Throttler.h"
```

Comment:
I figured that was the idea, but it is still tied to a specific config option for max sync and it dump status for number of syncs.